src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml

```
@@ -61,7 +61,38 @@ nginx_sites:
         # If the header is not provided, do not rate limit
         - key: '""'
           value: '""'
+    - name: "$restore_limit"
+      variable: "$days_since_last_sync"
+      var_mapping:
+        - key: "0"
+          value: 0
+        - key: "~\d+"
+          value: 1
+        - key: "default"
+          value: '""'  
+    - name: "$synced_today_bucket"
+      variable: "$restore_limit"
+      var_mapping:
+        - key: "0"
+          value: "$limit_bucket"
+        - key: "default"
+          value: '""'
+    - name: "$synced_old_bucket"
+      variable: "$restore_limit"
+      var_mapping:
+        - key: "1"
+          value: "$limit_bucket"
+        - key: "default"
+          value: '""'        
```

I think you can reduce this to:

```suggestion
    - name: "$synced_today_bucket"
      variable: "$days_since_last_sync"
      var_mapping:
        - key: "0"
          value: "$limit_bucket"
        - key: "default"
          value: '""'
    - name: "$synced_old_bucket"
      variable: "$days_since_last_sync"
      var_mapping:
        - key: "~[1-9]\d*"  # match number > 0
          value: "$limit_bucket"
        - key: "default"
          value: '""'        
```